frontend/lib/resolution/scope-queries.cpp

```diff
@@ -379,8 +379,17 @@ static bool doLookupInImports(Context* context,
       }
 
       if (named && is.kind() == VisibilitySymbols::SYMBOL_ONLY) {
-        result.push_back(BorrowedIdsWithName(is.scope()->id(), uast::Decl::PUBLIC));
-        found = true;
+        // Make sure the module / enum being renamed isn't private.
+        auto scopeAst = parsing::idToAst(context, is.scope()->id());
+        auto visibility = scopeAst->toDecl()->visibility();
+        auto foundIds =
```

Comment:
Do we have to introduce this abstraction? Could we just do `if (visibility == Decl::PUBLIC)` ... `createWithSinglePublicId`?

Reply:
Discussed this out of band, and decided to leave it as is (because `== PUBLIC` is not sufficient for the conditional, and the logic already exists in `BorrowedIdsWithName`).